Python-Input.md

```diff
@@ -0,0 +1,41 @@
+# Python Input() Function
+
+Many a times, in a program we need some input from the user. Taking inputs from the user makes the program feel a bit interactive. In Python 3, to take input from the user we have a function `input()`. Let's see some examples:
```

Comment:
~~Taking inputs from the user makes the program feel a bit interactive.~~ Taking inputs from the user makes the program feel interactive.